docs/src/main/asciidoc/developer-guide/swagger/core/overview.adoc

```diff
@@ -6,7 +6,7 @@
 
 === Version information
 [%hardbreaks]
-__Version__ : 1.44.0-SNAPSHOT
```

Comment:
What’s this about?

Reply:
Not entirely sure. Looks like the SNAPSHOT as a whole was upgraded to 2.0.0 somewhere along the line, but somehow these doc files were missed. mvnw install repeatedly causes this to happen after I reset the version to 1.44.0 in these files.